src/Utilities/functions.js

```diff
@@ -54,3 +55,59 @@ export const getSection = (key, schema = {}, storeSchema, success = () => {}) =>
         };
     }
 };
+
+export const concatApps = (apps = []) => {
+    return apps.reduce((acc, title, currentIndex) => (
+        `${acc}${title}${
+            currentIndex < apps.length - 1 ?
+                currentIndex < apps.length - 2 ? ',' : ' and'
```

Comment:
is there any possibility of using `join()` on the main array?

Reply:
Yeah, this might be a bit faster than the reduce in here.

Reply:
I had to change it to take empty and array with 1 element into account.